test/e2e/specs/dashboard/settings__site-visibility.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '../../lib/pw-base';
+
+test.describe( 'Dashboard: Site Visibility Settings', { tag: '@dashboard' }, () => {
+	test( 'As a new simple site user, I can set my site visibility to Private, so that only I can see my site', async ( {
+		helperData,
+		page,
+		pageDashboard,
+		pageIncognito,
+		siteNew,
+	} ) => {
+		const incognitoContentLocator = pageIncognito.locator( 'html' );
+
+		await test.step( 'Given I am on the site visibility settings page', async function () {
+			await page.goto(
+				helperData.getCalypsoURL(
+					`sites/${ siteNew.blog_details.site_slug }/settings/site-visibility`
+				)
+			);
+		} );
+
+		await test.step( 'When I set the site visibility to Private', async function () {
+			await pageDashboard.setSiteVisibility( 'Private' );
+		} );
+
+		await test.step( 'Then I can not see the site as an external visitor', async function () {
+			await pageIncognito.goto( siteNew.blog_details.url );
+			await expect.soft( incognitoContentLocator ).toContainText( 'Private Site' );
+		} );
+	} );
+
+	test( 'As a new simple site user, I can set my site visibility to Coming Soon, so that others see a nice coming soon message', async ( {
```

Comment:
By isolating each of the different visibility states into different tests, we've subtly changed what is being tested. New sites are "Coming soon" by default, so this test looks like it is changing a coming soon site to a coming soon site.

I think we want


```suggestion
	test( 'As a simple site user with a private site, I can set my site visibility to Coming Soon, so that others see a nice coming soon message', async ( {
```

Reply:
@p-jackson  As replied in a different comment, the new site created by the API is public, I have renamed the fixture to 'sitePublic` (the `new` is kind of superflous as using any fixture creates a new instance of it).
What do you think about splitting into `sitePublic`, `sitePrivate` and `siteComingSoon` and having relevant fixtures call common set up methods with an additional API call to set the public state? That way would set up a site exactly how we want then do something to it.